TestApps/TestApp/Player/VideoPlayer.swift

```diff
@@ -59,7 +59,7 @@ class VideoPlayer: AVPlayer {
     let QOEINFO_STARTUPTIME: Double = 2
     let QOEINFO_FPS: Double = 24
     let QOEINFO_DROPPEDFRAMES: Double = 10
-    let VIDEO_LENGTH: Double = 1800
+    let VIDEO_LENGTH: Double = 180
```

Comment:
what is the reason for changing this value?

Reply:
To match the actual length of the video used in the test app